### ISSUE TITLE
return type of fs

### ISSUE THREAD

**Issue by davidavdav** [dancasimiro/WAV.jl#26]
Hello,

Much more a feature request than an issue. 

I wonder if there is any particular reason to make the return type of `Fs` to be `Uint`.  I use WAV a lot for i/o, but I seem to be converting `Fs` to Floatingpoint all the time.    For one thing, Julia's default representation of Uint's (hexadecimal) isn't really handy in interactive use---I'd like to see 44100 or 16000 or whatever.  

Cheers,

**Comment by dancasimiro** [comment]
I think that the sample rate is stored as an unsigned int in the file, but It can be converted.

**Comment by davidavdav** [comment]
That is a good reason, then, I suppose.  WAV's can't store floating point sample rates then (say, very low frequency vibrations sampled at 0.1 Hz).  

I suppose I can always write my own wrapper that does the conversion of the 2nd argument.  Thanks.

**Comment by dancasimiro** [comment]
It is probably more convenient to do the conversion in wavread. I think that manually converting it is annoying too.

**Comment by davidavdav** [comment]
Agreed.  Another annoying thing is that `wavread` always returns a matrix, even if there is only a single channel in the file (which is most of the cases for me).  I guess this behaviour is a little reminiscent of Matlab's view on the world "everything is a matrix".  So in the end I usually have in the code
```julia
x, sr = wavread(wavfile)
sr = float(sr)
x = vec(mean(x, 2))
```
Which isn't particularly type-stable in `x` and `sr`.  (which might be inefficient---but I am not sure about that).

**Comment by davidavdav** [comment]
Perhaps a more julian thing to do is:
```julia
wavnorm(x) = vec(mean(x[1],2)), float(x[2]), x[3], x[4]
```
and open wavs (with my preferred interpretation) by
```julia
x, sr = wavread(filename) |> wavnorm
```

**Comment by dancasimiro** [comment]
Fixed by 18e714f
